usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -1550,6 +1550,16 @@ static CK_RV ab_unwrap_update_template(STDLL_TokData_t * tokdata,
         return CKR_TEMPLATE_INCONSISTENT;
     }
     for (i = 0; i < sizeof(attrs) / sizeof(CK_ATTRIBUTE); ++i) {
+        if (attrs[i].type == CKA_VALUE_LEN) {
+            switch (keytype) {
+            case CKK_GENERIC_SECRET:
```

Comment:
What's about CKK_DES2 and CKK_DES3? They show up in ep11_supported_mech_list()

Reply:
DES keys are fixed in length, and the length only depends on the key type (DES, DES2 or DES3). Thus, the PKCS#11 standard does not define the CKA_VALUE_LEN attribute for such keys.  The only key types where CKA_VALUE_LEN is defined per PKCS#11 standard is AES (i.e. 128, 192, 256) and generic secret.